discovery-provider/src/schema_track_simple.json

```diff
@@ -0,0 +1,278 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$ref": "#/definitions/Track",
+    "definitions": {
+        "Track": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "blockhash": {
```

Comment:
I think each property can also have `title`, `description`, and `example` [annotations](https://json-schema.org/understanding-json-schema/reference/generic.html) if we think that it'll be helpful per property.

Reply:
definitely! i didn't add those in for now just for simplicity sake. will do so in next commit